test/functorch/test_control_flow.py

```diff
@@ -578,22 +576,20 @@ def f(pred, x):
             grad_out = torch.ones_like(result)
             return torch.autograd.grad(result, (x,), grad_out)
 
-        gm = make_fx(f, tracing_mode="symbolic")(pred, x)
```

Comment:
The dispatching logic looks like this: in dynamo cond_op sees a functional_tensor[FakeTensor], then wrap_fx_proxy calls the cond_op tries to get its fake output, this triggers 1. autograd key (since some inputs require gradient), 2. functionalization key, then 3. fake_tensor_mode. For 1, we'll reenter_make_fx  fw and backward graph, which creates a new fake tesnor mode that's different from the fake tesnor mode on tensors (which is the dynamo fake tensor mode) and causing a mix fake tensor mode error.

Since the fix will make the tracing logic of autograd key even more hacky and we'll refactor the autorad key so that it doesn't materialize the graph soon, we just turn off symbolic tracing mode since it's not used anyway.